boards/sensebox_samd21/include/board.h

```diff
@@ -0,0 +1,169 @@
+ * 'XBEE1_EN_PIN'. To disable this bus call 'XBEE1_DISABLE', to re-enable it
+ * call 'XBEE1_ENABLE'.
+ * @{
+ */
+#define XBEE1_EN_PORT           PORT->Group[PB]
+#define XBEE1_EN_MASK           (1 << 3)
+#define XBEE1_EN_MODE           GPIO_OUT
+#define XBEE1_EN_PIN            GPIO_PIN(PB, 3)
+
+#define XBEE1_ENABLE            (XBEE1_EN_PORT.OUTCLR.reg = XBEE1_EN_MASK)
+#define XBEE1_DISABLE           (XBEE1_EN_PORT.OUTSET.reg = XBEE1_EN_MASK)
+
+#define XBEE1_CS_PIN            GPIO_PIN(PA, 18)
+
+#define XBEE1_INT_PIN           GPIO_PIN(PA, 21)
+/** @} */
+
+/**
+ * @name    XBEE2 bus
+ * @brief   XBEE2 is enabled by default. The enable pin is
+ * 'XBEE2_EN_PIN'. To disable this bus call 'XBEE2_DISABLE', to re-enable it
+ * call 'XBEE2_ENABLE'.
+ * @{
+ */
+#define XBEE2_EN_PORT           PORT->Group[PB]
+#define XBEE2_EN_MASK           (1 << 10)
+#define XBEE2_EN_PIN            GPIO_PIN(PB, 10)
+#define XBEE2_EN_MODE           GPIO_OUT
+
+#define XBEE2_ENABLE            (XBEE2_EN_PORT.OUTCLR.reg = XBEE2_EN_MASK)
+#define XBEE2_DISABLE           (XBEE2_EN_PORT.OUTSET.reg = XBEE2_EN_MASK)
+
+#define XBEE2_CS_PIN            GPIO_PIN(PA, 14)
+
+#define XBEE2_INT_PIN           GPIO_PIN(PA, 15)
+/** @} */
+
+/**
+ * @name    I2C bus
+ * @brief   I2C is enabled by default. The enable pin is
+ * 'I2C_EN_PIN'. To disable this bus call 'I2C_DISABLE', to re-enable it call
+ * 'I2C_ENABLE'.
+ * @{
+ */
+#define I2C_EN_PORT           PORT->Group[PB]
+#define I2C_EN_MASK           (1 << 11)
+#define I2C_EN_PIN            GPIO_PIN(PB, 11)
+#define I2C_EN_MODE           GPIO_OUT
+
+#define I2C_ENABLE            (I2C_EN_PORT.OUTSET.reg = I2C_EN_MASK)
+#define I2C_DISABLE           (I2C_EN_PORT.OUTCLR.reg = I2C_EN_MASK)
+/** @} */
+
+/**
+ * @name SX127X
+ * @brief SX127X configuration (on XBEE1 port). This particular board has
+ * merged DIO0 and DIO1 interupt pins into one (defined as DIOMULTI).
+ * @{
+ */
+#define SX127X_PARAM_SPI                    (SPI_DEV(0))
+
+#define SX127X_PARAM_SPI_NSS                XBEE1_CS_PIN        /* D23 */
+
+#define SX127X_PARAM_RESET                  GPIO_UNDEF
+
+#define SX127X_PARAM_DIOMULTI               XBEE1_INT_PIN       /* D24 */
+
+#define SX127X_PARAM_PASELECT               (SX127X_PA_RFO)
+
+#define SX127X_PARAMS                     { .spi       = SX127X_PARAM_SPI,     \
+                                            .nss_pin   = SX127X_PARAM_SPI_NSS, \
+                                            .reset_pin = SX127X_PARAM_RESET,   \
+                                            .dio0_pin  = SX127X_PARAM_DIO0,    \
+                                            .dio1_pin  = SX127X_PARAM_DIO1,    \
+                                            .dio2_pin  = SX127X_PARAM_DIO2,    \
+                                            .dio3_pin  = SX127X_PARAM_DIO3,    \
+                                            .paselect  = SX127X_PARAM_PASELECT }
+/** @} */
+
+/**
+ * @name    HDC1000 Temperature & Humidity sensor
```

Comment:
could use `@brief` here, as grouping is not required for a single `#define`, applies below for the TSL, too